scout/server/blueprints/variant/templates/variant/components.html

```diff
@@ -68,6 +68,11 @@
           {% else %}
             <span class="text-muted">BAM file(s) missing</span>
           {% endif %}
+          {% if variant.category == "str" %}
+            <div>
+              {{ reviewer_button(case,variant,case_groups,institute._id) }}
```

Comment:
Sure, why not - I'll have a look! It seemed like a good idea to treat them the same, but sure, if it's possible to disable, it seems good to disable it for real. Ideally we could keep it disabled for configured, but missing files, but not sure if we have the case info obj loaded with that right now. Let's see!

Reply:
Yeah, there seems to have been an intention to differentiate them with different hovers, but since hovers are pretty hard to get to work on disabled objects it doesn't really show easily. I removed the button if unconfigured, and kept it disabled if configured but files are missing.

Reply:
> Yeah, there seems to have been an intention to differentiate them with different hovers, but since hovers are pretty hard to get to work on disabled objects it doesn't really show easily. I removed the button if unconfigured, and kept it disabled if configured but files are missing.

OK!